pgrx/src/datum/json.rs

```diff
@@ -44,7 +44,7 @@ impl FromDatum for Json {
             let len = varsize_any_exhdr(varlena);
             let data = vardata_any(varlena);
             let slice = std::slice::from_raw_parts(data as *const u8, len);
-            let value = serde_json::from_slice(slice).expect("failed to parse Json value");
+            let value = serde_json::from_slice(slice).expect("failed to parse a json value");
```

Comment:
The reason it's called `expect` is that it should encode the _expectation_ instead of _what went wrong_ (because what went wrong is imperfect knowledge, because we often aren't checking and doesn't help the programmer... we want to give a _why_). We've been casual about this, but if we're going to spend time revising these:

Let's rephrase this as "Datum must refer to a JSON varlena" (if we've failed at parsing JSON, then it's not JSON, according to us).

Reply:
A follow up question then. Are the other guidelines then how to format messages there? In the standard library I saw that everything is lowercase and in some other crates too.

Reply:
That's because of https://rustc-dev-guide.rust-lang.org/diagnostics.html?highlight=style#diagnostic-output-style-guide

Reply:
Should `pgrx` follow it?

Reply:
It's usually good style to follow the stdlib, but I'm not that pressed over doing so.